sql/src/main/java/org/apache/druid/sql/SqlQueryPlus.java

```diff
@@ -66,6 +66,25 @@ public SqlQueryPlus(final String sql, final AuthenticationResult authResult)
     this(sql, (QueryContext) null, null, authResult);
   }
 
+  public SqlQueryPlus(
```

Comment:
nit: Should we have a Builder for this class considering that there are multiple constructors for it filling in `null` for different parameters?